apps/verticalhome/js/more_apps_navigation.js

```diff
@@ -4,7 +4,7 @@
   var MoreAppsNavigation = {
     items: null,
     selectedElemIndex: 0,
-
+    personalizeMode: false,
```

Comment:
May be it is more useful that this variable would be in app class not in MoreAppsNavigation class.